spec/version_spec.rb

```diff
@@ -33,6 +46,33 @@ def self.supported?
         end
       end
 
+      RSpec.describe Builderator::Control::Version do
+        before(:context) do
+          SCM.unregister(Test)
+          SCM.register(NoHistory)
+        end
+
+        after(:context) do
+          SCM.unregister(NoHistory)
+          SCM.register(Test)
+        end
+
+        context 'current' do
+          it 'provides an error message if no version is found' do
+            version_file = Util.relative_path('VERSION')
+            version_file.delete if version_file.exist?
+
+            expect(SCM.tags.last).to be_nil
+            expect { Version.current }.to raise_error RuntimeError
+          end
+
+          it 'falls back to VERSION file if no tags are found' do
+            Util.relative_path('VERSION').write('1.2.3')
```

Comment:
We've got [code in place already](https://github.com/rapid7/builderator/blob/fix-default-version/spec/spec_helper.rb#L8-L16) for `Util.relative_path` in tests. I'll see if I can extend it to skip writing the VERSION file to disk.